src/composables/multisigAccountCreate.ts

```diff
@@ -118,39 +127,38 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
    * First step of creating the multisig account
    * Prepare multisig account creation transaction
    */
-  async function prepareVaultCreationRawTx(payerId: string, accountId: string) {
-    if (!pendingMultisigCreationTxs.value[accountId]?.signedAttachTx) {
+  async function prepareVaultCreationRawTx(
+    payerId: Encoded.AccountAddress,
+    accountId: Encoded.AccountAddress,
+  ) {
+    if (pendingMultisigCreationTxs.value[accountId].signedAttachTx === undefined) {
```

Comment:
In this function you are referring to `pendingMultisigCreationTxs.value[accountId]` three times, what do you think of assigning it to variable before this line

```
const { signedAttachTx, rawTx } = pendingMultisigCreationTxs.value[accountId]
```